### ISSUE TITLE
[issue]: VTOY_LINUX_REMOUNT not working on Rocky Linux 9.5

### ISSUE THREAD

**Issue by teodor-pripoae** [ventoy/Ventoy#3035]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.99

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

MBR

### Disk Capacity

128GB

### Disk Manufacturer

Kingston HDD

### Image file checksum (if applicable)

None

### What happened?

I'm using Ventoy installed to a SSD to boot Rocky Linux ISOs. It worked just fine for months using Rocky 9.4 but now when Rocky 9.5 was released, the VTOY_LINUX_REMOUNT is no longer working.

My workflow:

- Mount ventoy to /mnt
- Download ISO to /mnt/my-iso-v1.x.x.iso
- Modify ventoy/ventoy.json and set `VTOY_DEFAULT_IMAGE` to the new ISO.
- umount /mnt
- reboot

The ISOs are made using `osbuild-composer` from standard Rocky Linux.

Now with Rocky Linux 9.5, I'm getting this error:

```
$ mount /dev/sde1 /mnt
mount: /mnt: /dev/sde1 already mounted or mount point busy.
```

I've tried booting in an identical image but based on Rocky 9.4 and it works fine to mount the ventoy partition.

**Comment by teodor-pripoae** [comment]
My ventoy.json:

```
{
    "control": [
	{ "VTOY_LINUX_REMOUNT": "1" },
        { "VTOY_MENU_TIMEOUT": "5" },
        { "VTOY_SECONDARY_TIMEOUT": "5" },
	{ "VTOY_DEFAULT_SEARCH_ROOT": "/" },
	{ "VTOY_DEFAULT_IMAGE": "/myiso-v1.0.130.iso" }
    ]
}
```

**Comment by teodor-pripoae** [comment]
Any updates ? I believe it's happening for other recent linux distros as well.

https://github.com/ventoy/Ventoy/issues/2928

**Comment by ventoy** [comment]
Please try latest Ventoy-1.1.01 and refer:  https://ventoy.net/en/doc_linux_remount.html

**Comment by teodor-pripoae** [comment]
I can't see the device in `/dev/mapper`. Do I need to reinstall ventoy from scratch ?

I've used:

```
 ./Ventoy2Disk.sh -u /dev/sdf
```

**Comment by teodor-pripoae** [comment]
I had to run `udevadm trigger` in order for it to show in `/dev/mapper`

```
[root@hostkey ventoy-1.1.01]# ls /dev/mapper
control
[root@hostkey ventoy-1.1.01]# dmsetup ls
sdf1    (253:1)
ventoy  (253:0)
[root@hostkey ventoy-1.1.01]# udevadm trigger
[root@hostkey ventoy-1.1.01]# ls /dev/mapper
control  sdf1  ventoy
```

**Comment by ventoy** [comment]
> I had to run `udevadm trigger` in order for it to show in `/dev/mapper`
> 
> ```
> [root@hostkey ventoy-1.1.01]# ls /dev/mapper
> control
> [root@hostkey ventoy-1.1.01]# dmsetup ls
> sdf1    (253:1)
> ventoy  (253:0)
> [root@hostkey ventoy-1.1.01]# udevadm trigger
> [root@hostkey ventoy-1.1.01]# ls /dev/mapper
> control  sdf1  ventoy
> ```

Can you finally mount it OK?

**Comment by teodor-pripoae** [comment]
Yeah, sometimes it does not appear in `/dev/mapper` after boot and I have to run `udevadm trigger`.

But I can mount it fine after.

**Comment by ventoy** [comment]
OK. I think this issue can be marked as fixed.
